Makefile

```diff
@@ -147,7 +147,7 @@ endif
 		-v $(CURDIR)/.coverage:/usr/src/github.com/vmware-tanzu/antrea/.coverage \
 		-v $(CURDIR):/usr/src/github.com/vmware-tanzu/antrea:ro \
 		-v /lib/modules:/lib/modules \
-		antrea/test test-integration $(USERID) $(GRPID)
+		antrea/test /bin/bash -c  "sysctl -w net.ipv6.conf.all.disable_ipv6=0 && test-integration" $(USERID) $(GRPID)
```

Comment:
What is this line for?

Reply:
When running unit test on a CentOS/MacOS host in container, some test will fail as IPv6 is disabled in container.  This line will enable IPv6 first, then start test-integration.

Reply:
you can use the `--sysctl net.ipv6.conf.all.disable_ipv6=1` command-line option with `docker run` instead

note that we are now skipping that specific integration test - the one that requires ipv6 - when ipv6 is disabled (https://github.com/vmware-tanzu/antrea/pull/1726). But if this is sufficient to get the test to systematically pass on macOS, I'm fine with this change. IIRC, setting this system parameter for the docker container won't affect the host, so no issue there.

Reply:
@hongliangl before considering this resolve, could you make the change I suggested and switch to `--sysctl net.ipv6.conf.all.disable_ipv6=1` if possible?

Reply:
`
@docker run --privileged --rm \
		-e "GOCACHE=/tmp/gocache" \
		-e "GOPATH=/tmp/gopath" \
		-e "INCONTAINER=true" \
		-w /usr/src/github.com/vmware-tanzu/antrea \
		-v $(DOCKER_CACHE)/gopath:/tmp/gopath \
		-v $(DOCKER_CACHE)/gocache:/tmp/gocache \
		-v $(CURDIR)/.coverage:/usr/src/github.com/vmware-tanzu/antrea/.coverage \
		-v $(CURDIR):/usr/src/github.com/vmware-tanzu/antrea:ro \
		-v /lib/modules:/lib/modules \
		--sysctl net.ipv6.conf.all.disable_ipv6=0 \
		antrea/test test-integration $(USERID) $(GRPID)
`
Is the above commands  as what you suggested?

Reply:
Why some test cases failed when ipv6 is disabled? We need to address it.

Reply:
IPv6 should be enabled in the container running integration test for this [test](https://github.com/vmware-tanzu/antrea/blob/b072436b0ba5ed3110968b9d960de9b5585f8063/test/integration/agent/route_test.go#L444) as it tests IPv6 route and neighbors. When I run the integration test on MacOS/CentOS7, this test failed, but it is ok on Ubuntu 18.04.  I found that IPv6 of container running on MacOS/CentOS7 is disabled by default, but it is enabled on Ubuntu 18.04.

Reply:
> Why some test cases failed when ipv6 is disabled? We need to address it.

IPv6 should be enabled in the container running integration test for this [test](https://github.com/vmware-tanzu/antrea/blob/b072436b0ba5ed3110968b9d960de9b5585f8063/test/integration/agent/route_test.go#L444) as it tests IPv6 route and neighbors. When I run the integration test on MacOS/CentOS7, this test failed, but it is ok on Ubuntu 18.04.  I found that IPv6 of container running on MacOS/CentOS7 is disabled by default, but it is enabled on Ubuntu 18.04.

Reply:
@hongliangl yes, that's what I suggested. LGTM